File: tests/nodejs/cases/case_252_container_handoff_global_05.js
/**
 * Task case 252: container_handoff_global_05 deep stress pattern.
 */

const retainedCache = new Map();
const retainedAudit = [];
const retainedHandlers = [];

function case252ContainerHandoffGlobal05(input) {
  const taskName = 'container_handoff_global_05';
  const raw = input || 'sample';
  const payload = {
    task: taskName,
    entity: 'extreme',
    stage: 'stress',
    input: raw,
    checksum: `${taskName}:${raw.length}`,
  };
  const box = { deep: { value: payload } };
  // ESCAPE: nested indirection retained globally.
  retainedCache.set('nested', box.deep.value);
  return 'ok';
}

module.exports = { case252ContainerHandoffGlobal05 };

File: tests/go/cases/case_252_container_handoff_global_05.go
package escape_tests

var retainedCase252 = []map[string]string{}

func Case252ContainerHandoffGlobal05(input string) string {
	raw := input
	if raw == "" {
		raw = "sample"
	}
	payload := map[string]string{
		"task": "container_handoff_global_05",
		"entity": "extreme",
		"stage": "stress",
		"input": raw,
		"checksum": "container_handoff_global_05:" + raw,
	}
	box := map[string]map[string]string{"v": payload}
	// ESCAPE: nested container handoff retained globally.
	retainedCase252 = append(retainedCase252, box["v"])
	return "ok"
}
